uber/configspec.ini

```diff
@@ -49,6 +49,11 @@ use_checkin_barcode = boolean(default=False)
 # registering, False to disallow promo codes.
 badge_promo_codes_enabled = boolean(default=False)
 
+# This controls whether the "Request Hotel Booking Info" step is including in
+# the preregistration workflow. True to include the "Request Hotel Booking
+# Info" step during preregistration, False to remove it.
+prereg_request_hotel_info_enabled = boolean(default=False)
```

Comment:
What do you think about making this a datetime config option? That way we can plan ahead for when this will turn off, and we don't have to restart the server to do it.